lighthouse/classes/event_properties/definitions/biosero/picked_samples_from_source.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Dict, Any
+from functools import cached_property
+from lighthouse.classes.event_properties.definitions import RunID, PlateBarcode
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract
+from lighthouse.classes.services.mongo import MongoServiceMixin
+from lighthouse.classes.services.cherrytrack import CherrytrackServiceMixin
+from lighthouse.constants.fields import FIELD_CHERRYTRACK_LH_SAMPLE_UUID, FIELD_EVENT_RUN_ID
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class PickedSamplesFromSource(EventPropertyAbstract, CherrytrackServiceMixin, MongoServiceMixin):
```

Comment:
Would `SamplesPickedFromSource` be clearer?

Reply:
done